kahuna/public/js/leases/leases.js

```diff
@@ -197,12 +197,12 @@ leases.controller('LeasesCtrl', [
             ctrl.updateLeases();
         });
 
-        $scope.$watch(() => ctrl.images.length, () => {
+        $scope.$watchCollection(() => Array.from(ctrl.images), () => {
+            ctrl.totalImages = ctrl.images.size;
             ctrl.updateLeases();
         });
 
         ctrl.resetLeaseForm();
-        ctrl.updateLeases();
```

Comment:
did you mean to remove this line?

Reply:
Sorry, should have added a comment to the PR 😊. Yep, this deletion was intentional.

`$scope.$watchCollection` will ... watch ... `ctrl.images` and when it changes, call `ctrl.updateLeases`.

Explicitly calling `updateLeases` again here amounts to it being called twice on component mount*, so isn't necessary.

⏩ 🚗 🐎 

*not sure what the Angular equivalent of React mounting is...